python/TestHarness/testers/Tester.py

```diff
@@ -214,56 +254,69 @@ def checkRunnableBase(self, options, checks):
       for x in self.specs[check]:
         if x[0] == '!':
           if inverse_set:
-            return (False, "(Multiple Negation Unsupported)")
+            reason = '(Multiple Negation Unsupported)'
```

Comment:
Hmm, reasons probably shouldn't have parenthesis in them. When we print them out the parenthesis should be added at that point.